build.sbt

```diff
@@ -20,6 +20,11 @@ sparkComponents ++= Seq("core", "streaming", "sql", "hive", "streaming-kafka", "
 
 parallelExecution in Test := false
 
+coverageHighlighting := {
+  if (scalaBinaryVersion.value == "2.10") false
```

Comment:
As you noticed, Scoverage hits weird AST processing issues under Scala 2.10. These are fixed in 2.11, where we're able to re-enable coverage highlighting.